### ISSUE TITLE
How to prepare our own data for inference?

### ISSUE THREAD

**Issue by kaushikCanada** [open-mmlab/mmdetection3d#60]
For example I want to use this to detect objects on the toronto-3d data which has point wise labels but not ready for object detection. This data is in the form of ply files. What do i need to do to get objects from this data using this library?
Thanks in advance.

**Comment by Tai-Wang** [comment]
In theory you can organize your data like one of datasets supported by our codebase (KITTI, nuScenes, Lyft, scannet, sunrgbd), generate its corresponding info with data converter and use a pretrained model to test on your own dataset. However, currently, domain gap always exists between the data obtained by different LiDARs in different scenes. So the results may be not very satisfactory. If you can access detection ground truth and finetune on your own dataset, a better result can be expected.
